src/api/app/models/staging/staging_project.rb

```diff
@@ -0,0 +1,141 @@
+module Staging
+  class StagingProject < Project
+    has_many :staged_requests, class_name: 'BsRequest', foreign_key: :staging_project_id, dependent: :nullify
+    has_many :status_reports, through: :repositories, inverse_of: :checkable
+    belongs_to :staging_workflow, class_name: 'Staging::Workflow'
+
+    def staging_identifier
+      name[/.*:Staging:(.*)/, 1]
+    end
+
+    def untracked_requests
+      requests_to_review - staged_requests
+    end
+
+    # The difference between staged requests and requests to review is that staged requests are assigned to the staging project.
+    # The requests to review are requests which are not related to the staging project (unless they are also staged).
+    # They simply need a review from the maintainers of the staging project.
+    def requests_to_review
+      @requests_to_review ||= BsRequest.with_open_reviews_for(by_project: name)
+    end
+
+    def building_repositories
+      set_buildinfo unless @building_repositories
+      @building_repositories
+    end
+
+    def broken_packages
+      set_buildinfo unless @broken_packages
+      @broken_packages
+    end
+
+    def missing_reviews
```

Comment:
no test code for missing_reviews, which we could use to help in the next refactoring iteration @bgeuken ?

Reply:
Right. As stated before this method is going to change. I was just waiting for some feedback. So I paused writing the tests for this part.